src/components/ImageView/index.tsx

```diff
@@ -148,13 +126,10 @@ function ImageView({isAuthTokenRequired, url, fileName, onError}) {
         }
     };
 
-    /**
-     * @param {SyntheticEvent} e
-     */
     const trackPointerPosition = useCallback(
-        (e) => {
+        (e: MouseEvent) => {
             // Whether the pointer is released inside the ImageView
-            const isInsideImageView = scrollableRef.current.contains(e.nativeEvent.target);
+            const isInsideImageView = scrollableRef.current?.contains(e.target as Node);
```

Comment:
Please test if this change won't create regressions

Reply:
One way to solve this:

First, add this `ReactMouseEvent` type to `react` import:

```ts
import React, {MouseEvent as ReactMouseEvent, useCallback, useEffect, useRef, useState} from 'react';
```

Then, use it like this:

```ts
    const trackPointerPosition = useCallback(
        (e: MouseEvent) => {
            const mouseEvent = e as unknown as ReactMouseEvent;
            // Whether the pointer is released inside the ImageView
            const isInsideImageView = scrollableRef.current?.contains(mouseEvent.nativeEvent.target as Node);

            if (!isInsideImageView && isZoomed && isDragging && isMouseDown) {
                setIsDragging(false);
                setIsMouseDown(false);
            }
        },
        [isDragging, isMouseDown, isZoomed],
    );
```